packages/cms/schemas/objects/pageCard.js

```diff
@@ -62,5 +62,19 @@ export default {
       initialValue: 'dark',
       validation: (rule) => rule.required(),
     },
+    {
+      name: 'backgroundColor',
+      title: 'Background Colour',
+      description:
+        'The entered value should be in any valid HEX format, e.g. #EF5632, #ef5632, #EF5632A1, #ef5632a1, #ffe, #ffea',
+      type: 'string',
+      initialValue: '#FFF',
+      validation: (Rule) =>
+        Rule.custom((input) =>
+          /^#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{4}){1,2}$/i.test(input)
```

Comment:
Be good to include the reference to where you found this! ☺️

Reply:
@joshuaellis it's coming from here: https://stackoverflow.com/questions/8027423/how-to-check-if-a-string-is-a-valid-hex-color-representation, but it's not exactly that, I adjusted it because I could not find one that would include alpha channels.
Where shall I add it? In the code above as a comment?

Reply:
Yep!!